packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/EthERC20Bridge.sol

```diff
@@ -190,8 +192,14 @@ contract EthERC20Bridge is L1Buddy {
         uint256 maxGas,
         uint256 gasPriceBid
     ) external payable onlyIfConnected returns (uint256) {
-        string memory name = SafeERC20Namer.tokenName(erc20);
-        string memory symbol = SafeERC20Namer.tokenSymbol(erc20);
+        string memory name = "";
+        string memory symbol = "";
+        try ERC20(erc20).name() returns (string memory _name) {
```

Comment:
Should we use https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/extensions/IERC20Metadata.sol

This way we keep interfaces all around for the calls. Also ERC20 is a name-clash with the StandardTokenType enum.

Reply:
Ah seems like it's still unreleased https://github.com/OpenZeppelin/openzeppelin-contracts/blob/74e03de7604ca99c2f8e476476a6dc362779c356/CHANGELOG.md#unreleased